maintenance/deleteBatch.php

```diff
@@ -39,6 +39,9 @@ public function __construct() {
 		$this->addOption( 'i', "Interval to sleep between deletions" );
 		$this->addArg( 'listfile', 'File with titles to delete, separated by newlines. ' .
 			'If not given, stdin will be used.', false );
+
+		// Wikia change
+		$this->addOption( 'by-id', 'Provided list contains article IDs instead of titles', false, false /* $withArgs */ );
```

Comment:
It's actually `separated by newlines` and that's what documentation for `listfile` says.